package.json

```
@@ -10,7 +10,7 @@
     "lint": "eslint ."
   },
   "files": [
-    "contexts/*.json",
+    "contexts/",
```

I'd be more explicit here to avoid packing extra files:
```suggestion
    "contexts/**/*.json",
```

---

Updated [27869ba](https://github.com/credential-handler/vc-examples/pull/16/commits/27869ba6c0def6804b90abb3f3103f406f98d013).